src/renderers/shared/fiber/ReactFiberDevToolsHook.js

```diff
@@ -33,9 +33,35 @@ if (
     onCommitFiberUnmount,
   } = __REACT_DEVTOOLS_GLOBAL_HOOK__;
 
-  injectInternals = function(internals: Object) {
+  injectInternals = function(
+    findFiberByHostInstance: (instance: any) => Fiber,
+    findHostInstanceByFiber: (component: Fiber) => any,
+    bundleType: number,
```

Comment:
I think `bundleType` or `version` can be calculated here instead of being passed as arguments.